index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello World CR</title>
+    <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+    <p>Hello Microverse!</p>
```

Comment:
- You should make use of the `<h1>`  tag so as to make your code more semantic. it is also required in the curriculum requirements
- You should follow the [Github flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)